src/main/java/uk/nhs/digital/nhsconnect/nhais/service/edifact_to_fhir/FP69FlagRemovalTransactionMapper.java

```diff
@@ -0,0 +1,35 @@
+package uk.nhs.digital.nhsconnect.nhais.service.edifact_to_fhir;
+
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.StringType;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.PersonName;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.ReferenceTransactionType;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Transaction;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.nhais.model.fhir.ParameterNames;
+import uk.nhs.digital.nhsconnect.nhais.model.fhir.ParametersExtension;
+
+import java.util.List;
+
+@Component
+public class FP69FlagRemovalTransactionMapper implements FhirTransactionMapper {
+
+    @Override
+    public void map(Parameters parameters, Transaction transaction) {
+        mapFreeText(parameters, transaction);
+    }
+
+    private void mapFreeText(Parameters parameters, Transaction transaction) {
+        var nhsIdentifier = transaction.getPersonName()
+            .map(PersonName::getNhsNumber)
+            .flatMap(FhirTransactionMapper::mapToNhsIdentifier)
```

Comment:
The same optional comment as here
https://github.com/nhsconnect/integration-adaptor-nhais/pull/168/files#r456267417